### DISCUSSION TITLE
Welcome to clustercode Discussions!

### DISCUSSION THREAD

**Discussion by ccremer** [ccremer/clustercode#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by phil-colossus** [comment]
Hi Chris,

I have a challenge with ffmpeg and I've found your Clustercode software which may help, but before I start building PCs I would appreciate your thoughts.

As a radio amateur I transmit digital TV in real time, my setup is, streaming RTMP from OBS into ffmpeg, ffmpeg compresses the stream with h264 or h265, then ffmpeg converts it to an IPTS and streams it to my TV transmission equipment, all in realtime and it works fine.

The latest compression h266 is now available for ffmpeg and I would like to use it for realtime TV, the problem is h266 is very processor intensive and there is no way I can can compress h266 in realtime  on my i7 PC.

My thoughts are to use 4 x i5 PCs as a cluster to share the ffmpeg processing, that way being able to stream h266 in realtime due to the the extra processing power that a cluster would have, but I had no idea if this is possible, until I found your Clustercode.

Simple question, will your Clustercode do the job?

Phil

**Reply by ccremer** [reply]
Hi Phil

Clustercode was initially intended to convert files in the background, not for realtime input/output.
It literally takes file from storage, splits it up, converts, and merges them back together in the storage. All in Kubernetes.

Currently I would have no idea how to make that to real-time streaming. The current architecture wasn't built for that.
Also, atm the software is more alpha than being usable.

For the time being, unfortunately the answer is rather No for realtime processing.

Chris
